rules/windows/process_creation/win_susp_replace_lolbin.yml

```diff
@@ -0,0 +1,25 @@
+title: Ingress Tool Transfer Using Replace.exe
+id: 6ccf0c00-1061-4195-a724-6d9c0058b036
+description: Detect Download operations using Replace.exe.
+status: experimental
+references:
+  - https://lolbas-project.github.io/lolbas/Binaries/Replace
+author: Jonhnathan Ribeiro, oscd.community
+date: 2020/10/07
+tags:
+  - attack.command_and_control
+  - attack.t1105
+logsource:
+  category: process_creation
+  product: windows
+detection:
+  selection:
+    Image|endswith:
+      - '\replace.exe'
+    CommandLine|contains|all:
+      - "\\\\\\\\"
```

Comment:
Hello @w0rk3r !

You need to change this line to the `'\\\'` if you want to detect '\\' in the event.

Let me explain. 
This is what the converter produces with the current quotes and amount of slashes:

```
CommandLine.keyword:*\\\\\*
```

This is what the converter will produce if you will change the line to the `'\\\'`:

```
CommandLine.keyword:*\\\\*
```

And that is exact query that will find the '\\' in events:

<img width="1147" alt="Screenshot 2020-10-09 at 02 19 53" src="https://user-images.githubusercontent.com/13090109/95527688-4f1ec500-09d6-11eb-8529-33df914e581e.png">

Reply:
Hey @yugoslavskiy,

I'm using Splunk in my Lab, and from the translation, I obtain the following query:

![image](https://user-images.githubusercontent.com/26856693/95581495-83fb3e00-0a0f-11eb-8753-eac81bc65068.png)

That matches correctly in Splunk:

![image](https://user-images.githubusercontent.com/26856693/95581592-a2f9d000-0a0f-11eb-9a76-52b5b6534c23.png)

The results of conversion to ElasticSearch seems correct too:

![image](https://user-images.githubusercontent.com/26856693/95581667-c755ac80-0a0f-11eb-87c5-c5ceb2d92cdf.png)

Am I missing something?

Reply:
Most probably, the problem here is that the original sigmac (fully updated from master branch) process the rule in a different way, production the following:

```
tools/sigmac -t es-qs -c tools/config/elk-windows.yml /tmp/lol.yml
(Image.keyword:(*\\replace.exe) AND CommandLine.keyword:*\\\\\* AND CommandLine.keyword:*\/A*)
```

the uncoder, for some reason, produces a different result. I don't know why, but I would rather develop a rule that would work with original sigma tools, rather than with 3rd party convertors.

what do you think?

Reply:
Hey @w0rk3r! It seems that this PR was merged while we were discussing the rule :D Could you please get back to this discussion? (:

Reply:
Hey @yugoslavskiy, I'm afk till Sunday, I will get back into this when I get back

Reply:
Just reviewed on sigma, and this representation doesn't make sense for me, but as it's the officially supported by sigmac, i changed the rule and opened a new PR